File: use-cases/merchant-fulfillment/code/java/src/main/java/lambda/utils/NotificationOrderSummary.java
package lambda.utils;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class NotificationOrderSummary {

    @JsonProperty("OrderStatus")
    public String orderStatus;

    @JsonProperty("FulfillmentType")
    public String fulfillmentType;
}

File: use-cases/solicitations/code/csharp/src/sp-api-csharp-app/lambda/utils/NotificationOrderSummary.cs
namespace SpApiCsharpApp.lambda.utils
{
    public class NotificationOrderSummary
    {       
        public String OrderStatus { get; set; }

        public String FulfillmentType { get; set; }

        public String MarketplaceId { get; set; }

        public String EarliestDeliveryDate { get; set; }

        public String LatestDeliveryDate { get; set; }



    }
}